server/lib/routes/get-index.js

```diff
@@ -35,15 +40,25 @@ module.exports = function (config) {
   route.path = '/';
 
   route.process = function (req, res) {
-    var flowEventData = flowMetrics(FLOW_ID_KEY, req.headers['user-agent']);
+    var userAgent = req.headers['user-agent'];
+    var flowEventData = flowMetrics(FLOW_ID_KEY, userAgent);
 
-    res.render('index', {
+    // Render options
+    var renderOptions = {
       config: serializedConfig,
       flowBeginTime: flowEventData.flowBeginTime,
       flowId: flowEventData.flowId,
       // Note that staticResourceUrl is added to templates as a build step
       staticResourceUrl: STATIC_RESOURCE_URL
-    });
+    };
+
+    // Display Firefox iOS install banner if on compatible device
+    var agent = uaParser.parse(userAgent);
+    if (agent && agent.os.family === 'iOS') {
```

Comment:
we should probably avoid calling `.parse` every time on that route and just the `<meta` tag everywhere. Yelp does this.